CONTRIBUTING.md

```diff
@@ -164,21 +474,32 @@ New committers are approved by majority vote of the TSC ([see TSC charter](https
         * Use the imperative mood in the subject line
         * Wrap the body at 72 characters
         * Use the body to explain what and why vs. how
-     
-- Example commit message
-
-```
-Fix OOM caused by foo in bar
-
-Foo was pack ratting ByteBuffers causing OOM.
-
-Cherry-pick of c2bf3b0 (https://github.com/foo/bar/pull/123/commits/c2bf3b025ff456921c4c5ff4bcb814740e823061)
-
-Co-authored-by: Foo Bar <foo@bar.com>
-```
+    * Ensure all code is peer reviewed within your own organization or peers before submitting
+    * Implement and/or address existing feedback before requesting further review
+    * Make a good faith effort to locate example or referential code before requesting someone else direct you towards it
+    * If you see a lack of documentation, create a separate commit with draft documentation to fill the gap
+        * This documentation can be iterated on same as any code commit, demonstrate in real time that you are learning the code section
+    * Implement or modify relevant tests, otherwise provide clear documentation why test updates were not necessary
```

Comment:
```suggestion
    * Implement or modify relevant tests, otherwise provide clear explanation why test updates were not necessary
```
Suggest "explanation" to avoid using "documentation" in a way different than the (user-facing) documentation being discussed in the immediately previous lines, to reduce ambiguity.